charts/aws-vpc-cni/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-vpc-cni
 version: 1.1.17
```

Comment:
Is the version already bumped up from v1.11.0?

Reply:
Yes, it was. We didn't release the previous v1.11.1

> The chart version is already updated as previous chart was not released.

Reply:
The previous one is 1.1.16 (https://github.com/aws/amazon-vpc-cni-k8s/blob/v1.11.0/charts/aws-vpc-cni/Chart.yaml#L3) and current one is 1.1.17

Reply:
And yeah, cni-metrics-helper/chart is updated too - https://github.com/orsenthil/amazon-vpc-cni-k8s/blob/update_release_manifests_1.12/charts/cni-metrics-helper/Chart.yaml#L18

Previous one is version: 0.1.8 and current one is 0.1.9

Reply:
I don't see the diff here in this PR for charts/cni-metrics-helper. Also `appVersion` needs to be bumped to 1.11.2

Reply:
This is very strange to me. The `ag` command didn't include it.

```
[senthilx@88665a371033:~/go/src/github.com/aws/amazon-vpc-cni-k8s]$ ag v1.11.1 -l
test/go.sum
CHANGELOG.md
config/master/cni-metrics-helper-cn.yaml
config/master/aws-k8s-cni-us-gov-west-1.yaml
config/master/cni-metrics-helper-us-gov-west-1.yaml
config/master/aws-k8s-cni.yaml
config/master/cni-metrics-helper-us-gov-east-1.yaml
config/master/aws-k8s-cni-cn.yaml
config/master/cni-metrics-helper.yaml
config/master/aws-k8s-cni-us-gov-east-1.yaml
charts/aws-vpc-cni/Chart.yaml
charts/aws-vpc-cni/values.yaml
[senthilx@88665a371033:~/go/src/github.com/aws/amazon-vpc-cni-k8s]$ cd charts/
/Users/senthilx/go/src/github.com/aws/amazon-vpc-cni-k8s/charts
[senthilx@88665a371033:~/go/src/github.com/aws/amazon-vpc-cni-k8s/charts]$ ag v1.11.1 -l
cni-metrics-helper/Chart.yaml
cni-metrics-helper/values.yaml
aws-vpc-cni/Chart.yaml
aws-vpc-cni/values.yaml
```

I will update.

Reply:
Okay. I see that. It is because of exclusion of cni-metrics-helper in the gitignore.

```
$ cat .gitignore
aws-k8s-agent
aws-cni
verify-aws
verify-network
*~
*.swp
.idea/
*.iml
.DS_Store
portmap
grpc-health-probe
cni-metrics-helper
coverage.txt
build/
vendor
```

Reply:
Let's update the .gitignore and remove the cni-metrics-helper separately.

```
The following paths are ignored by one of your .gitignore files:
charts/cni-metrics-helper
```

The version of cni-metrics-helper chart is now updated.